bolt/lib/Rewrite/RewriteInstance.cpp

```diff
@@ -4568,6 +4568,26 @@ void RewriteInstance::updateELFSymbolTable(
     }
   }
 
+  // Add runtime library start and fini address symbols
+  if (RuntimeLibrary *RtLibrary = BC->getRuntimeLibrary()) {
+    auto AddSymbol = [&](StringRef Name, uint64_t Address) {
+      if (!Address)
+        return;
+
+      ELFSymTy NewSymbol;
+      NewSymbol.st_shndx = ELF::SHN_ABS;
```

Comment:
Is there a reason you make this an `ABS` symbols instead of linking it to the correct section?

Reply:
If you can suggest an easy way to do this without overcomplicating the code.. The hot/cold start symbols below are done like this, since it is just a label I don't really think there is a difference anyway